src/harness/reference_models/common/mpool.py

```diff
@@ -105,8 +111,12 @@ def GetNumWorkerProcesses():
 
 
 def _partial_fn(fn):
+  # sleep to avoid returning too fast so each worker
+  # gets one job in the RunOnEachWorkerProcess.
+  time.sleep(0.5)
```

Comment:
should we consider using a random number 0.5 - 1.0 to be more reliable ?

Reply:
Not really in this case. This is just for the use of the little function nbelow which insure that you run  one function on each pool.
This is only used in the `test_main.py` script which allows to configure each pool geo data drivers for example.